main.go

```diff
@@ -306,17 +304,20 @@ func getKeyStore(datastore string, keyName string) *Keystore {
 	return NewKeystore(dirstore, keyName)
 }
 
-func loadTenantToken(datastore string) ([]byte, error) {
+func loadTenantToken(config *menderConfig, datastore string) ([]byte, error) {
+
 	dirstore := NewDirStore(datastore)
-	raw, err := dirstore.ReadAll(defaultTenantTokenFile)
+
+	raw, err := dirstore.ReadAll(config.GetTenantToken())
```

Comment:
Actually `TenantToken` is now the actual token, and not a file or a path, but I guess stopping now, and rehashing the details here is a good idea.
@kacf thoughts?